src/components/NewsLetter.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+    <section class="newsletter">
+        <img src="https://images.pexels.com/photos/1109543/pexels-photo-1109543.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=650&w=940">
```

Comment:
Can you move the img source to a computed property or prop instead of having this hardcoded please.

Reply:
Ok make the changes accordingly